src/libNode/MicroBlockPreProcessing.cpp

```diff
@@ -1226,11 +1528,24 @@ unsigned char Node::CheckLegitimacyOfTxnHashes(bytes& errorMsg) {
            TXN_DS_TARGET_NUM)) {
     vector<TxnHash> missingTxnHashes;
     if (!VerifyTxnsOrdering(m_microblock->GetTranHashes(), missingTxnHashes)) {
-      LOG_GENERAL(WARNING, "The leader may have composed wrong order");
+      if (!m_prePrepRunning) {
+        LOG_GENERAL(WARNING, "The leader may have composed wrong order");
+      }
       return LEGITIMACYRESULT::WRONGORDER;
     }
 
     if (missingTxnHashes.size() > 0) {
+      if (!m_prePrepRunning) {
+        LOG_GENERAL(WARNING,
```

Comment:
Maybe better to say "Unexpected missing txns during prepare phase - rejecting microblock"